suite-common/metadata-types/src/metadataTypes.ts

```diff
@@ -239,3 +239,6 @@ export interface MetadataState {
 
 export type OAuthServerEnvironment = 'production' | 'staging' | 'localhost';
 export type MetadataEncryptionVersion = 1 | 2;
+
+// todo:
+export type PasswordEntry = any;
```

Comment:
Can you be more specific?

Reply:
Ok but extending the comment would be nice

Reply:
done e322a42b24dacf25f6f99206c1ac93cf42be0cd3